CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,64 @@
+# Code of Conduct for Lyra
+
+While we have specific guidelines for various tools (see links below), in general, you should:
+
+-   **Be nice**: Be courteous, respectful and polite to fellow community members. No offensive comments related to gender, gender identity or expression, sexual orientation, disability, physical appearance, body size, race, religion; no sexual images in public spaces, real or implied violence, intimidation, oppression, stalking, following, harassing photography or recording, sustained disruption of talks or other events, inappropriate physical contact, or unwelcome sexual attention will be tolerated.
+-   **Encourage diversity and participation**: Make everyone in our community feel welcome, regardless of their background, and do everything possible to encourage participation in our community.
+-   **Keep it legal**: Basically, don't get us in trouble. Share only content that you own, do not share private or sensitive information, and don't break the law.
+-   **Stay on topic**: Make sure that you are posting to the correct online channel and avoid off-topic discussions. Also remember that nobody likes spam.
+
+Guideline violations --- 3 strikes method
+-----
+
+The point of this section is not to find opportunities to punish people, but we do need a fair way to deal with people who do harm to our community. Extreme violations of a threatening, abusive, destructive, or illegal nature will be addressed immediately and are not subject to 3 strikes.
+
+1.  **First occurrence**: We'll give you a friendly, but public, reminder that the behavior is inappropriate according to our guidelines.
+2.  **Second occurrence**: We'll send you a private message with a warning that any additional violations will result in removal from the community.
+3.  **Third occurrence**: Depending on the violation, we might need to delete or ban your account.
+
+Notes:
+
+-   Obvious spammers are banned on first occurrence. If we don’t do this, we’ll have spam all over the place.
+-   People who are committing minor formatting / style infractions will get some education, rather than hammering them in the 3 strikes process.
+-   Contact lyra@puppet.com to report abuse or appeal violations. 
+
+Activity-specific guidelines
+-----
+
+-   [Getting technical help](#getting-technical-help)
+-   [Slack guidelines](#irc-guidelines)
```

Comment:
Kill this line, since there's no Slack section below

Reply:
Fixed